### DISCUSSION TITLE
Place Filter in the column like searchable?

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#8879] [Help]
Is it possible to place a filter in the searchable column like: 

```
->searchable(isIndividual: true, isGlobal: false)
```

I would like to place a Select Field instead of the textInput where normaly the searchable field appears? Or maybe is it possible to change the searchable field to select field?

**Comment by leandrocfe** [comment]
Isn't possible I think. You could use [select-filters](https://filamentphp.com/docs/3.x/tables/filters#select-filters)  / [displaying-filters-above-the-table-content](https://filamentphp.com/docs/3.x/tables/filters#displaying-filters-above-the-table-content)

**Comment by VladimirBazhenov** [comment]
+1 for this functionality

**Comment by KuenzelIT** [comment]
+1

This would be great for big tables, where there are many filters above the table and the mapping to columns can be difficult.

**Comment by leek** [comment]
<img width="2558" height="858" alt="image" src="https://github.com/user-attachments/assets/24c2edad-5085-4dc4-84aa-7ce53c21b9f5" />

https://github.com/leek/filament-header-filters
